locale/fr/about/working-groups.md

```diff
@@ -0,0 +1,250 @@
+* La gestion de la promotion des intérevenants en conférence et meetups.
+* La gestion de la promotion des événements communautaires.
+* La publication régulière de résumés de mises à jour et tout autre contenu
+  promotionnel.
+
+### [Docker](https://github.com/nodejs/docker-node)
+
+L'objectif du Groupe de Travail Docker est de construire, maintenir et améliorer
+les images Docker officielles du projet Node.js.
+
+Ses responsabilités incluent:
+* Maintenir à jour les images officielles Docker en lien avec les sorties Node.js.
+* Choisir et implémenter des améliorations et/ou des corrections sur les images.
+* Maintenir et améliorer la documentation des images.
+
+### [Addon API](https://github.com/nodejs/nan)
+
+Le Groupe de Travaill Addon API est responsable de la maintenance du projet NAN
+et du package _nan_ correspondant sur npm. Le projet NAN met à disposition
+une couche d'abstraction pour les auteurs d'add-on natifs pour Node.js,
+en aidant à l'écriture de code compatible avec les versions actives de Node.js,
+V8 et libuv.
+
+Ses responsabilités incluent:
+* Maintenir le dépôt GitHub [NAN](https://github.com/nodejs/nan), y compris le
+  code, les tickets et la documentation.
+* maintenir le dépôt GitHub [addon-examples](https://github.com/nodejs/node-addon-examples)
+  y compris le code, les tickets et la documentation.
+* Maintenir l'API pour Addon C++ du projet Node.js, en coordination avec le TSC
+  de Node.js
+* Maintenir la documentation des Addon du projet Node.js, en coordination avec
+  le TSC de Node.js
+* Maintenir le package _nan_ sur npm, et publier de nouvelles versions si besoin.
+* Communiquer sur le futur de Node.js et de l'interface NAN afin de prévenir
+  la communauté à l'avance d'éventuels changements.
+
+La liste des membres actuels du groupe se trouve sur leur
+[README](https://github.com/nodejs/nan#collaborators).
+
+### [Benchmarking](https://github.com/nodejs/benchmarking)
+
+L'objectif du Groupe de Travail Benchmark est d'arriver à un
+consensus sur une ensemble de points de référence qui pourront
+être utilisés pour:
+* suivre et communiquer sur les gains de performance entre chaque
+  version de Node.js
+* éviter les régressions de performances entre les versions
+
+Ses responsabilités incluent:
+* Identifier un ou plusieurs points de référence qui reflètent
+  les utilisations client.
+  Celà incluera certainement plusieurs points pour couvrir
+  les usages habituels de Node.js, y compris la réduction de latence
+  et la concurrence élevée
+* Travailler pour obtenir un consensus communautaire sur la liste
+  choisie
+* Aider à l'exécution régulière des benchmarks choisis sur les builds
+  Node.js
+* Traquer/Communiquer sur les performances des différentes versions
+
+### [Post-mortem](https://github.com/nodejs/post-mortem)
+
+Le Groupe de Travail de Diagnostic Post-Mortem est dédié à la maintenance
+et à l'amélioration du débuggage portmortem pour Node.js. Il cherche
```

Comment:
> débogage post-mortem

ugly... I know...

Reply:
Il also read somewhere "déverminage"... Can't we just stick to "débug"? :D Just joking...